src/scripts/types/ApplicationTypes.js

```diff
@@ -32,7 +33,8 @@ export type RootState = {
   uploader: UploaderRecord,
   user: UserRecord,
   videos: VideoRecordMap,
-  player: PlayerRecord
+  player: PlayerRecord,
+  notifications: NotificationRecord
```

Comment:
Is this correct? Are we sure it is not `Array<NotificationRecord>`? Can there really be only one notification at once?

Reply:
yes you are totally right, this is an `Array<NotificationRecord>`